azurerm/internal/services/cosmos/common/schema.go

```diff
@@ -37,3 +40,56 @@ func MongoCollectionAutoscaleSettingsSchema() *schema.Schema {
 
 	return autoscaleSettingsDatabaseSchema
 }
+
+func CosmosDbIndexingPolicySchema() *schema.Schema {
+	return &schema.Schema{
+		Type:     schema.TypeList,
+		Optional: true,
+		Computed: true,
+		MaxItems: 1,
+		Elem: &schema.Resource{
+			Schema: map[string]*schema.Schema{
+				// `automatic` is excluded as it is deprecated; see https://stackoverflow.com/a/58721386
+				"indexing_mode": {
+					Type:             schema.TypeString,
+					Optional:         true,
+					Default:          documentdb.Consistent,
+					DiffSuppressFunc: suppress.CaseDifference, // Open issue https://github.com/Azure/azure-sdk-for-go/issues/6603
+					ValidateFunc: validation.StringInSlice([]string{
+						string(documentdb.Consistent),
+						string(documentdb.None),
+					}, false),
+				},
+
+				"included_path": {
+					Type:     schema.TypeList,
+					Optional: true,
+					Computed: true,
+					Elem: &schema.Resource{
+						Schema: map[string]*schema.Schema{
+							"path": {
```

Comment:
I only did `path` here as the `indexes` field was accepted by the REST API, but did not return the `indexes` field in response.